go/cmd/vttlstest/vttlstest.go

```diff
@@ -17,143 +17,118 @@ limitations under the License.
 package main
 
 import (
-	"flag"
-	"fmt"
-	"io"
-
-	"github.com/spf13/pflag"
+	"github.com/spf13/cobra"
 
 	"vitess.io/vitess/go/exit"
-	"vitess.io/vitess/go/vt/log"
 	"vitess.io/vitess/go/vt/logutil"
 	"vitess.io/vitess/go/vt/tlstest"
-
-	// Include deprecation warnings for soon-to-be-unsupported flag invocations.
-	_flag "vitess.io/vitess/go/internal/flag"
 )
 
-var doc = `
-vttlstest is a tool for generating test certificates and keys for TLS tests.
-
-To create a toplevel CA, use:
-  $ vttlstest --root /tmp CreateCA
-
-To create an intermediate CA, use:
-  $ vttlstest --root /tmp CreateIntermediateCA servers
-
-To create a leaf certificate, use:
-  $ vttlstest --root /tmp CreateSignedCert --parent servers server
+var (
+	root       = "."
+	parent     = "ca"
+	serial     = "01"
+	commonName = ""
+
+	rootCmd = &cobra.Command{
+		Use:   "vttlstest",
+		Short: "vttlstest is a tool for generating test certificates and keys for TLS tests.",
+		Long:  "vttlstest is a tool for generating test certificates and keys for TLS tests.",
+	}
 
-To get help on a command, use:
-  $ vttlstest <command> --help
-`
+	createCACmd = &cobra.Command{
+		Use:   "CreateCA",
+		Short: "Create certificate authority",
+		Long:  "Create certificate authority",
+		Args:  cobra.NoArgs,
+		Run:   runCreateCA,
+	}
 
-type cmdFunc func(subFlags *flag.FlagSet, args []string)
+	createIntermediateCACmd = &cobra.Command{
+		Use:   "CreateIntermediateCA",
```

Comment:
typically you include the full usage in the `Use` line, so like `CreateIntermediateCA <ca name>` or similar

Reply:
(there's a bunch of examples of this in the `vtctldclient` command tree